src/App.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from "react";
+import BookList from "./components/BookList/BookList";
+import styles from "./App.module.scss";
+
+const App = () => {
+	//setting state for the books
+	const [booksData, setBooksData] = useState([]);
+	//setting state for the search result
+	const [search, setSearch] = useState("");
+
```

Comment:
add another piece of state to display an error message if no books found:  `const [error, setError] = useState(false);`